huaweicloud/resource_huaweicloud_ecs_instance_v1.go

```diff
@@ -373,9 +380,14 @@ func resourceEcsInstanceV1Read(d *schema.ResourceData, meta interface{}) error {
 
 func resourceEcsInstanceV1Update(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*config.Config)
-	computeClient, err := config.ComputeV2Client(GetRegion(d, config))
+	computeV2Client, err := config.ComputeV2Client(GetRegion(d, config))
 	if err != nil {
-		return fmtp.Errorf("Error creating HuaweiCloud compute client: %s", err)
+		return fmtp.Errorf("Error creating HuaweiCloud compute v2.1 client: %s", err)
```

Comment:
suggest to remove `HuaweiCloud` and can use fmt.Errorf directly

Reply:
will do it later